### ISSUE TITLE
useTransition & others: use the Values API?

### ISSUE THREAD

**Issue by weaverryan** [stimulus-use/stimulus-use#129]
Hi again!

I was trying to use the `transition` behavior's controller class directly today without writing any custom JavaScript. It is, of course, totally possible to do this :). Except that the options - e.g. `hiddenClass` or `transitioned` don't appear to have a way to set them via HTML attributes. 

Could we add a way to set these via HTML attributes? And I would love for it to use the Values API. In the Symfony world (where I come from), we have a helper that makes setting values on a controller *dead* simple and clean.

And while it *is* currently possible to use data attributes to set things like `enter-active`, I think these should also be settable via the values API. We don't need to remove the current way of setting them as data attributes, just add the ability to use options :).

WDYT? Thanks!

**Comment by adrienpoly** [comment]
Yes exactly, I have a WIP branch on exactly this topic. And this should be generalised to other mixins too.
